python/CMakeLists.txt

```diff
@@ -650,27 +675,38 @@ endif()
 
 # Acero
 if(PYARROW_BUILD_ACERO)
-  if(PYARROW_BUNDLE_ARROW_CPP)
-    bundle_arrow_lib(${ARROW_ACERO_SHARED_LIB} SO_VERSION ${ARROW_SO_VERSION})
-    if(MSVC)
-      bundle_arrow_import_lib(${ARROW_ACERO_IMPORT_LIB})
+  list(APPEND CYTHON_EXTENSIONS _acero)
```

Comment:
Could you move this to the end of this block like the original code does?

https://github.com/apache/arrow/blob/890916306ce4ca799ac027f5bde4fe70eace7b27/python/CMakeLists.txt#L653-L661